### ISSUE TITLE
Initial pixel polishes

### ISSUE THREAD

**Issue by catherinecarter** [phetsims/number-pairs#14]
These are initial pixel polishes:

- [x] Default the Addends checkbox to 'on' 
- [x] Default "Total Jump" to 'on'
- [x] For the location-based representations (on Intro and Twenty screens), the blue section should be on the right to minimize visual dissonance between the other representations (since the cubes, kittens, and number line all have blue on the right)
- [x] The sentence might work better left aligned rather than centered. The sentences bounces when changing addends
- [x] Animate the organization of objects like in Number Play
- [x] Update Equation accordion box size to:
   - [x] horizontally be the same distance from the number bond as the Sentence accordion box
   - [x] become vertically shorter to accommodate more vertical padding between it and the checkboxes on the Number Line representation
- [x] Make the Sentence accordion box a fixed width (vertically changing size dynamically is great, as long as there's room for the dual language toggle)
- [x] Change kitten icon in bottom radio button group to be the blue cat - the yellow is difficult to see
- [x] Make the number line slightly smaller inside the accordion box to padding between the left addend and the edge of the counting area:   
![image](https://github.com/user-attachments/assets/ed39d68f-f543-4eaa-a1ff-1ae8050c722e)

- [x] Round corners of all accordion boxes to have a 5 pixel radius
- [x] Move accordion box collapse/expand button to be 10 pixels from top and left edge   
![image](https://github.com/user-attachments/assets/e8275e08-44d3-425c-809c-cdfae94a647f)
- [x] Number line representation: Change zero distance loop to have the arrow stay centered on dot    
![image](https://github.com/user-attachments/assets/d38067ac-d12d-405c-b6fc-f2afcde6a84c)

**Comment by marlitas** [comment]
> Move accordion box collapse/expand button to be 10 pixels from top and left edge

Let's chat about this. The mockups show a button layout when the accordion box is open and a different one when it is closed. I can't do both and what you're asking me to do would affect the way the title appears as well... maybe you're okay with that but I want to make sure.

**Comment by catherinecarter** [comment]
Thanks, Marla. I think the 10 pixels came from a "standard" I thought was true. But maybe I remembered wrong and it's 5 pixels? Whatever makes sense so the button is in the same place when open or closed. I checked a few sims and there seems to be a different number of pixels from the top/bottom/left edge when closed, so let's talk and decide on a good distance so it doesn't create a weird UI. Here are some examples: 
- Projectile Data Lab:   <img width="137" alt="image" src="https://github.com/user-attachments/assets/67ffb4b1-3437-447c-ba6b-87bf94022440">
- Buoyancy: Basics:  ![image](https://github.com/user-attachments/assets/79576008-aaeb-46b4-b50e-d90e83eedd32)
- Calculus Grapher: ![image](https://github.com/user-attachments/assets/7f9da50e-40ea-4dbc-a6d0-1ddaf6b5409e)
- Quadrilateral:  ![image](https://github.com/user-attachments/assets/3c221e7b-6ac3-4835-abc3-89605e025690)
